workflow_templates.tf

```diff
@@ -30,6 +30,11 @@ locals {
           { run = "terragrunt apply --terragrunt-non-interactive -no-color -input=false -compact-warnings -auto-approve $PLANFILE " }
         ]
       }
+      import = {
```

Comment:
if we're doing import, let's do other commands as well like `state rm`?

Reply:
yeah, I'll add that